meta/regex.rb

```diff
@@ -17,3 +17,17 @@
   mutation '/#{nil}n/'
   mutation '/#{self}n/'
 end
+
+Mutant::Meta::Example.add :regexp do
+  source 'true if /foo/'
+
+  singleton_mutations
+  mutation 'false if /foo/'
+  mutation 'true if //'
+  mutation 'nil if /foo/'
+  mutation 'true if true'
+  mutation 'true if false'
+  mutation 'true if nil'
+  mutation 'true if /a\A/'
```

Comment:
This is an odd regexp to generate. I don't think it can ever match anything. Was that intentional?